### PR TITLE
Update azure cosmos db for java from track1 to track2

### PR DESCRIPTION
**Pull request by v-hongli1**
1. Update azure cosmos db for Java from track1 to track2
2. Change file:
  1). pom.xml
  2). src/main/java/com/microsoft/azure/management/cosmosdb/samples/CreateCosmosDBWithIPRange.java
  3). src/main/java/com/microsoft/azure/management/samples/Utils.java

@scottaddie, @joshfree, @kyle-patterson, @mayurid and @aayush3011 for notification.

### CODE REVIEW

**Comment by v-xuto** [comment]
@scottaddie, @joshfree, @kyle-patterson, @mayurid and @aayush3011  Could you please review this PR?

**Comment by v-xuto** [comment]
@scottaddie, @joshfree, @kyle-patterson, @mayurid and @aayush3011 Could you please review this PR?

**Comment by joshfree** [comment]
@kushagraThapar could you route this to someone on your team?

**Comment by v-xuto** [comment]
@scottaddie, @joshfree, @kyle-patterson, @mayurid, @kushagraThapar and @aayush3011 Could you please review this PR?

**Review by joshfree — APPROVED** [review]

File: pom.xml [hunk 1]
```diff
@@ -50,6 +50,21 @@
...
   <dependencies>
+    <dependency>
+      <groupId>com.azure.resourcemanager</groupId>
+      <artifactId>azure-resourcemanager</artifactId>
+      <version>2.9.0</version>
+    </dependency>
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-identity</artifactId>
+      <version>1.4.0</version>
```
Comment (joshfree) [inline pom.xml]: given how old the PR is now, it would be good to refresh this to much newer versions of libraries.

Comment (v-hongli1) [reply pom.xml]: According to your suggestion, fixed the newer version of libraries in the new version.

**Comment by v-xuto** [comment]
@joshfree We have updated by your comments, please re-review this PR.

**Comment by v-xuto** [comment]
@joshfree We have updated by your comments, please re-review this PR. Thanks a lot.

**Comment by v-xuto** [comment]
@joshfree We have updated by your comments, please re-review this PR. Thanks a lot.
If you have no questions about this PR, please approve. After that, we will merge this PR.

**Comment by v-xuto** [comment]
@joshfree We have updated by your comments, please re-review this PR. Thanks a lot.
If you have no questions about this PR, please approve. After that, we will merge this PR.

**Comment by v-xuto** [comment]
@joshfree We have updated by your comments, please re-review this PR. Thanks a lot.
If you have no questions about this PR, please approve. After that, we will merge this PR.

**Comment by v-xuto** [comment]
@joshfree We have updated by your comments, please re-review this PR. Thanks a lot.
If you have no questions about this PR, please approve. After that, we will merge this PR.

**Review by joshfree — APPROVED** [review]

**Comment by v-xuto** [comment]
@jongio Could you help merge this PR? Thanks a lot.

**Pull request merged**